2-2-3.py

```diff
@@ -0,0 +1,31 @@
+'''
+На вход программе подается строка текста из натуральных чисел. Из элементов строки формируется список чисел.
+Напишите программу, которая меняет местами соседние элементы списка (a[0] c a[1], a[2] c a[3] и т.д.).
+Если в списке нечетное количество элементов, то последний остается на своем месте.
+
+Формат входных данных
+На вход программе подается строка текста, содержащая натуральные числа, разделенные пробелами.
+
+Формат выходных данных
+Программа должна вывести измененный список, разделяя его элементы одним пробелом.
+
+'''
+
+
+inputString = input().split(' ')
+nums, outputList, outputList_, resultList = [], [], [], []
+
+for i in inputString:
```

Comment:
1. `i` again!
2. no need to convert a string to a list of chars in your case - you can access string chars by an index in the same way as in a list.